rabbitmq/conf.yaml.example

```diff
@@ -60,6 +60,8 @@ instances:
     # using the aliveness API. If you prefer only certain vhosts to be monitored
     # with service checks then you can list the vhosts you care about.
     #
+    # This also impact the metric 'rabbitmq.connections'.
```

Comment:
could we "integrate" this more in the existing doc? I'm thinking of something like:

```     
     # Service checks and `rabbitmq.connections` metric:
     # By default a list of all vhosts is fetched and each one will be checked
     # using the aliveness API and a `connections` metric.
     # If you prefer only certain vhosts to be monitored with service checks
     # and the number of `connections` then you can list the vhosts you care about.
```